### DISCUSSION TITLE
Exclude "$RECYCLE.BIN" from thumbnail creation?

### DISCUSSION THREAD

**Discussion by boredofchoosingnames** [outdoorbits/little-backup-box#207] [General]
Hi,

Just setup and starting to learn this system. Seems really go so far. Need to work out where my speed bottle neck is, but thanks for the hard work.

My question is, how do exclude folders from the thumbnail creation? LBB has just scanned 9000+ files that windows helpfully left in the "$RECYCLE.BIN" folder that I have tried to delete.

Sorry if I have missed something, but I couldn't see a way to define the folders to scan.

Regards,

Frank

**Comment by outdoorbits** [comment]
It's no problem. I'll exclude this folder within the next few days. Then you just can update using the GUI.

**Reply by boredofchoosingnames** [reply]
Fantastic. Great work.

F

**Comment by outdoorbits** [comment]
Instant help would be to use the LBB integrated file browser and delete the trash.

**Reply by boredofchoosingnames** [reply]
Thanks, that's what I did on this occaision. I'm sure WIndows will leave me a present in the future though.

F

**Reply by boredofchoosingnames** [reply]
So I think I worked out why this happened in the first place. Becuse the SD card adaptor was set to read only, the copy was done to the root of the storage USB. When it started to read images to create the thumbnails, it then read all the folders in there, not just the DCIM, so it went through the trash folder too.

Does that sound likely? It did the same with a .Trash-1000 folder as well.

I may have caused a problem that wasn;t there. Apologies.

I'll monitor it and let you know if it happens again.

On the brightside, my screen arrived yesterday and that is working well, so I t can track prgress better.

F

**Comment by outdoorbits** [comment]
Should work now. Would you please test it against your windows $recycle.bin and give feedback?

**Reply by boredofchoosingnames** [reply]
I'll let you know, currently I have deleted the folder so will try and set up a test.

F

**Comment by boredofchoosingnames** [comment]
Oh dear, I seem to have replied to myself, orginal message so you don;t miss it:
So I think I worked out why this happened in the first place. Becuse the SD card adaptor was set to read only, the copy was done to the root of the storage USB. When it started to read images to create the thumbnails, it then read all the folders in there, not just the DCIM, so it went through the trash folder too.

Does that sound likely? It did the same with a .Trash-1000 folder as well.

I may have caused a problem that wasn;t there. Apologies.

I'll monitor it and let you know if it happens again.

On the brightside, my screen arrived yesterday and that is working well, so I t can track prgress better.

F

**Comment by outdoorbits** [comment]
LBB checks all the target device for images without thumbnails. It's not focused on the last backups subdirectory. This has different reasons. The check itself is running very fast.
So your first idea sounds more likely to me.

**Reply by boredofchoosingnames** [reply]
Thanks. Seems to be getting to grips with this.
It has not read the  $recycle.bin again, but has read another format, for example the "/202306091608-22352/.Trash-1000/files".

This folder is not visible in Tiny File Manager", Windows or linux. I think it is a trash folder created by action cameras (or maybe their linux sub-system). I have examples for a Yi4+ and Gopro6, which is what I intend to back up with this system in the future.

Anyway, thanks for replying.

F

**Comment by boredofchoosingnames** [comment]
*correction* I can see these in folders in Linux. Interesting to know they are there as it would allow recoverry of accidental deletes, but probably one to omit for thumbnails?

**Comment by outdoorbits** [comment]
Ok, this shouldn't be a problem. I'll add this folder pattern to the banned list tonight.

**Comment by outdoorbits** [comment]
Please let me know about other names of trash folders if you find some. I always backup directly from camera and only specified folders so I never got these folders.

**Comment by boredofchoosingnames** [comment]
Great stuff thank you.
I think we have Linux and Windows trash covered. I'm afraid I dont know what MacOs does as don't have one.

**Comment by outdoorbits** [comment]
Could you please update and test against your trash-1000 folder? Google says, mac uses ~/.Trash as trash folder.

**Reply by boredofchoosingnames** [reply]
Morning. I had a chance to test this yesterday. From what I can see, LBB copied all the files which is good. It also ignored the images I put sepcicically in the trash/refuse folders so this seems to have worked okay.

I will keep an eye on it in real world use and comment if it is not working. Will be testing the setup abroad for the first time this week.

thanks again.

F
